changelogs/fragments/solaris-setfacl-chmod-fallback.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - become - work around setfacl not existing on modern Solaris (and possibly failing on some filesystems even when it does exist)
```

Comment:
These could be combined into a single file unless you really want them to be separate.

Reply:
Intentionally separate files and commits because I wasn't sure if the new Solaris fallback was backportable to 2.11 or not. It's "kind of" a feature, allowing the "new" way to set ACLs on tmpdirs? But also kind of a bug fix?